src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php

```diff
@@ -31,15 +35,16 @@ protected function configure(): void
         ;
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $expirationTime = $this->getContainer()->getParameter('sylius_order.cart_expiration_period');
+        $expirationTime = $this->container->getParameter('sylius_order.cart_expiration_period');
         $output->writeln(sprintf(
             'Command will remove carts that have been idle for <info>%s</info>.',
             (string) $expirationTime
         ));
 
-        $expiredCartsRemover = $this->getContainer()->get('sylius.expired_carts_remover');
+        /** @var ExpiredCartsRemoverInterface $expiredCartsRemover */
+        $expiredCartsRemover = $this->container->get('sylius.expired_carts_remover');
         $expiredCartsRemover->remove();
 
         return 0;
```

Comment:
While we're here, we should use constants for this `Command::SUCCESS`

Reply:
@Roshyo I think a dedicated PR for this one should be better.

Reply:
I agree about new PR doing that, also IMHO it's just "syntax sugar" that doesn't give any advantage.